src/Util/LocaleCodeProvider.php

```diff
@@ -86,4 +100,35 @@ public function reset(): void
     {
         $this->cache = [];
     }
+
+    private function getSupportedLocales(): array
+    {
+        $filePath = __DIR__ . '/../Resources/config/locales.json';
+        if (!file_exists($filePath)) {
+            $this->logger->error('Locales file not found: ' . $filePath);
+            return [];
+        }
+
+        $localesJson = file_get_contents($filePath);
```

Comment:
In 3rd party extensions we ask devs to not use direct access to the file system. Are we allowed to do it? If not, I'll find another way.

Reply:
We could write the locales into a PHP file to load them via class loader. Have a look at `CrawlWebhookEventNamesCommand`. `file_get_contents` will fail if e.g. s3 is connected